INSTALL.md

```diff
@@ -83,12 +83,14 @@ your `.zshrc`.
     git clone https://github.com/zsh-users/zsh-syntax-highlighting.git ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-syntax-highlighting
     ```
 
-2. Activate the plugin in `~/.zshrc`:
+2. Activate the plugin **at the bottom** of `~/.zshrc`:
```

Comment:
This could be understood to mean "Append `plugins=( "${plugins[@]}" zsh-syntax-highlighting )` to your zshrc".

I think the right instruction would be to ensure that zsh-syntax-highlighting is last in the array and that the command that reads the array is run at the end of zshrc, but none of the maintainers uses omz so we rely on patch submissions in this regard.